doc/man3/SSL_CTX_set1_curves.pod

```diff
@@ -96,6 +100,9 @@ is -1.
 When called on a client B<ssl>, SSL_get_shared_group() has no meaning and
 returns -1.
 
+SSL_get_negotiated_group() returns the NID of the negotiated group on a
```

Comment:
@mspncp Any suggestions on how to reword this?

Reply:
As a new _public_ API, would this be better returning a string?  I.e. `OBJ_nid2sn` on the NID.

Reply:
@paulidale I feel returning NID as integer is fine for this API, as application can call `OBJ_nid2sn` again to get text format.